lesson04/assignment/css/_components.content.scss

```diff
@@ -0,0 +1,268 @@
+    margin: 0.35rem;
+    animation: dotPreload 1.2s ease-in-out infinite;
+
+    &:nth-of-type(1) {
+      animation-delay: 0;
+    }
+
+    &:nth-of-type(2) {
+      animation-delay: 150ms;
+    }
+
+    &:nth-of-type(3) {
+      animation-delay: 300ms;
+    }
+
+    &:nth-of-type(4) {
+      animation-delay: 450ms;
+    }
+  }
+}
+@keyframes dotPreload {
+  0% {
+    bottom: 0;
+    background-color: $stormviolet;
+  }
+
+  15% {
+    background-color: $stormviolet;
+  }
+
+  20% {
+    bottom: -.5rem;
+  }
+
+  30% {
+    background-color: $stormviolet;
+  }
+
+  40% {
+    bottom: 0.75rem;
+  }
+
+  45% {
+    background-color: $cyan;
+    box-shadow: 0 0 8px $cyan;
+  }
+
+  80% {
+    background-color: $stormviolet;
+  }
+
+  100% {
+    bottom: 0;
+    background-color: $stormviolet;
+  }
+}
+/* =============CORE RAIN ANIMATION============= */
+.drop {
+  position: absolute;
+  bottom: 100%;
+  width: 15px;
+  height: 120px;
+  pointer-events: none;
+  animation: drop 0s linear infinite;
+}
+@keyframes drop {
+  0% {
+    transform: translateY(0vh);
+  }
+
+  100% {
+    transform: translateY(100vh);
+  }
+}
+
+.stem {
+  width: 2px;
+  height: 60%;
+  transform: rotate(15deg);
+  background: linear-gradient(to bottom, rgba(255, 255, 255, 0), rgba(255, 255, 255, 0.25));
+  animation: stem 0s linear infinite;
+}
+@keyframes stem {
+  0% {
+    opacity: 1;
+  }
+
+  65% {
+    opacity: 0.75;
+    background: $grey;
+  }
+
+  70% {
+    opacity: 0.5;
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+/* =============RAIN ELEMENTS============= */
+.rainContainer {
+  position: absolute;
+  height: 65vh;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  &__front {
+    position: relative;
+    display: block;
+    width: 100%;
+    height: 65vh;
+    opacity: 0.5;
+    /* border: 2px dotted green; */
+  }
+
+  &__back {
+    position: absolute;
+    display: block;
+    width: 90%;
+    height: 65vh;
+    bottom: 20vh;
+    /* right: 2vw; */
+    /* border: 2px dotted yellow; */
+    opacity: 0.25;
+  }
+}
+/* ==========MAIN SITE CONTENT========== */
+.main {
+  height: 65vh;
+  width: 100%;
+  margin: auto;
+  position: relative;
+  z-index: 2;
+  border: 2px solid magenta;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main__map {
+  width: 10rem;
+  height: 10rem;
+  border: 2px solid green;
+  background-color: green;
+  z-index: 2;
+}
+
+.main__list {
+  width: 10rem;
+  height: 10rem;
+  color: white;
+  border: 2px solid purple;
+}
+
+.sidenav {
+  display: block;
+  position: absolute;
+  overflow: hidden;
+  background: $grey;
+  color: $white;
+  font-size: 16px;
+  border: none;
+  border-radius: 0 0.8rem 0.8rem 0;
+  height: auto;
+  width: auto;
+  padding: 1rem;
+  margin: 1rem 1rem 1rem 0;
+  transition: all 0.4s cubic-bezier(.67,-.51,.42,1.43);
+  z-index: 11;
+
+  &:hover {
+    font-size: 24px;
```

Comment:
Remember, we always want to size fonts relatively, with rem or % unit.

Reply:
D'oh! Thanks Cheri!